kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -787,4 +787,14 @@
     });
   }
 
+  var draftbutton = document.getElementsByClassName('btn-draft')[0];
```

Comment:
Vanilla-JS DOM access is better written as `var draftButton = document.querySelector('.btn-draft');`